kafka-test/src/test/java/io/specmesh/kafka/ClientsFunctionalDemoTest.java

```diff
@@ -54,20 +55,19 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 class ClientsFunctionalDemoTest extends AbstractContainerTest {
     private static final KafkaApiSpec apiSpec = new KafkaApiSpec(getAPISpecFromResource());
-    private AdminClient adminClient;
-    private SchemaRegistryClient schemaRegistryClient;
+    private final AdminClient adminClient;
```

Comment:
Consider adding a negative test, where user has valid credentials, but should not be allowed to:
 1. produce
 2. consume
... from the topic

(Otherwise, all tests would still pass even if either no, or open access, acls where being set)

Or maybe this sort of testing would be better done in the other functional testing.

Reply:
agree - I have that in the KafkaAPI functional tests - this class is more for demo purposes